### ISSUE TITLE
Command + ` do not work on Mac OSX

### ISSUE THREAD

**Issue by ignaciolg** [alacritty/alacritty#4172]
Alacritty installed with brew, default configuration. 

I'm able to use different commands like `Command + N` to create new windows, but the `Command + /`` one does nothing.  

### System
OS: macOS Catalina 10.15.6
Version: alacritty 0.5.0 (a6681e3)

### Logs
[alacritty.key.log](https://github.com/alacritty/alacritty/files/5130930/alacritty.key.log)

Keyboard and bindings: `alacritty --print-events`

**Comment by ignaciolg** [comment]
Sorry, I think that I didn't explain the issue in the right way. 

I want to use the default OS behavior (the one defined on the system Preferences >  Keboard ... screen). 

I.e: I want to be able to, from an Alacritty windows, focus the next one by pressing Command + `.

In the logs I can see the key events, so probably Alacritty is capturing them, preventing the OS to execute the default action. Is that possible?

**Comment by chrisduerr** [comment]
If it's an OS behavior, then there's nothing we can do to override it. The hierarchy is pretty clear with things like that and if it's bound to the OS, we will never find out about that.

> I.e: I want to be able to, from an Alacritty windows, focus the next one by pressing Command + `.

Do you mean focusing the next window in general, or the next Alacritty window?

**Comment by nixpulvis** [comment]
> Do you mean focusing the next window in general, or the next Alacritty window?

I believe the expectation for <kbd>Command</kbd> + <kbd>`</kbd> is to cycle through the current application's windows.

This issue is probably dependent on #607.

**Comment by chrisduerr** [comment]
Closing as duplicate of #607 then. Currently with all Alacritty windows being different applications, it's impossible for macOS to support this. Once that's fixed, this should be fixed too.
